### ISSUE TITLE
'maintained node versions' referring pennding Node version

### ISSUE THREAD

**Issue by y-ogura** [browserslist/browserslist#311]
Nuxt.js build error, because of 'maintained node versions' referring pennding Node version.

show this error message.
```
Module build failed (from ./node_modules/nuxt/node_modules/postcss-loader/src/index.js):
BrowserslistError: Unknown version 11 of Node.js
```
![2018-10-23 12 50 15](https://user-images.githubusercontent.com/30112982/47334583-5271f680-d6c2-11e8-8ae6-17fb85cb3e02.png)

**Comment by sibiraj-s** [comment]
<img width="1411" alt="screenshot 2018-10-23 at 11 02 16 am" src="https://user-images.githubusercontent.com/20282546/47337795-200ccd00-d6b3-11e8-9a38-77735d79c022.png">

**Comment by dangreen** [comment]
All my CI builds fall with this error.

**Comment by CosticaPuntaru** [comment]
a ugly hack to allow you to work on localhost:

node_modules/browserslist/index.js:678
![image](https://user-images.githubusercontent.com/6418405/47344064-39f8e080-d6b0-11e8-8338-505be0b24498.png)

**Comment by andreasradloff** [comment]
Changing `maintained node versions` to `node 10` in your .browserslistrc works as a temp fix.

**Comment by ai** [comment]
Oh. I waked up and need 15 minutes for breakfast. Then I will investigate the source of the error and how to fix it.

**Comment by dangreen** [comment]
@ai https://github.com/browserslist/browserslist/pull/312

**Comment by ai** [comment]
Fixed in 4.3.2
